se/Mortgage/v0/schema/definitions/Applicant.yaml

```diff
@@ -0,0 +1,185 @@
+"$id": "https://open-broker.org/se/Mortgage/v0/schema/definitions/Applicant"
+"$schema": "http://json-schema.org/draft-07/schema#"
+
+title: Applicant
+description: |
+  Describe the attributes of an applicant, such as housing situation,
+  employment details, financial information, marital status and citizenships.
+
+type: object
+additionalProperties: false
+required:
+  - ssn
+  - employmentStatus
+  - employmentStatusSinceYear
+  - employmentStatusSinceMonth
+  - dependentChildren
+  - paymentRemarks
+  - housingType
+  - housingCostPerMonth
+  - monthlyIncome
+  - maritalStatus
+  - citizenships
+  - countriesOfResidence
+  - taxResidentOf
+
+properties:
+  ssn:
+    type: string
+    title: '12-digit Social Security Number (sv. Personnr) with'
```

Comment:
Remove "with"

Reply:
It took 8 months before someone noticed :laughing: